src/main/target/common_defaults_post.h

```diff
@@ -20,6 +20,10 @@
 
 // pg/max7456
 
+#ifndef DEBUG_MODE
```

Comment:
Not sure of the value of this change, since it will become pointless as soon as we move to unified targets.